src/neuroglancer/datasource/dvid/base.ts

```diff
@@ -21,6 +22,7 @@ export class DVIDSourceParameters {
 }
 
 export class VolumeChunkSourceParameters extends DVIDSourceParameters {
+  volumeType: VolumeType;
```

Comment:
Actually, for now you could even avoid an explicit encoding parameter, and just determine the encoding based on the value of spec.compressedSegmentationBlockSize